airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/AbstractJdbcDestination.java

```diff
@@ -84,11 +87,26 @@ public AirbyteConnectionStatus check(final JsonNode config) {
     }
   }
 
+  /**
+   * This method is deprecated. It verifies table creation, but not insert right to a newly created
+   * table. Use attemptSQLCreateAndDropTableOperations with isNeedTryMakeTestDataInsertInTable
+   * argument instead
+   */
+  @Deprecated
   public static void attemptSQLCreateAndDropTableOperations(final String outputSchema,
                                                             final JdbcDatabase database,
                                                             final NamingConventionTransformer namingResolver,
                                                             final SqlOperations sqlOps)
       throws Exception {
+    attemptSQLCreateAndDropTableOperations(outputSchema, database, namingResolver, sqlOps, false);
+  }
+
+  public static void attemptSQLCreateAndDropTableOperations(final String outputSchema,
+                                                            final JdbcDatabase database,
+                                                            final NamingConventionTransformer namingResolver,
+                                                            final SqlOperations sqlOps,
+                                                            final boolean isNeedTryMakeTestDataInsertInTable)
```

Comment:
The variable `isNeedTryMakeTestDataInsertInTable` is difficult to understand what purpose it serves. Also if a method is deprecated shouldn't that method also be made `private` to not allow users to call that method unintentionally? 

It may help to just have a javadoc comment here that describes each variable and the intent of `attemptSQLCreateAndDropTableOperations` to help others get up to speed quicker

Reply:
Updated naming to make for explicit and extended javadocs as discussed. Thanks